File: pinyougou_sellergoods_interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import java.util.List;
import java.util.Map;

import com.pinyougou.pojo.TbBrand;

import entity.PageResult;

public interface BrandService {

	public  List<TbBrand>  findAll();

	public PageResult findPage(int pageNum,int pageSize);

	public void add(TbBrand brand);

	public TbBrand findOne(Long id);

	public void update(TbBrand brand);

	public void dele(Long[] ids);

	public PageResult search(int pageNum, int pageSize, TbBrand brand);

	public List<Map> findBrandList();
}

File: pinyougou_manager_web/src/main/webapp/js/service/brandService.js
contentapp.service("brandService",function($http){
	
//	  分页查询
	  this.findByPage=function(pageNum,pageSize){
		 return $http.get("../brand/findPage/"+pageNum+"/"+pageSize);
	  }
	  
	  this.findAll=function(){
		  return $http.get("../brand/findAll");
	  }
	  
	  this.findBrandList=function(){
		  return $http.get("../brand/findBrandList");
	  }
	  
	  
	  this.add=function(entity){
		return  $http.post("../brand/add",entity);
	  }
	  
	  this.update=function(entity){
		return  $http.post("../brand/update",entity);
	  }
	  
	  
//	  根据id查询对象
	  this.findOne=function(id){
		  return $http.get("../brand/findOne/"+id);

	  }
	  
	 

	  this.dele=function(selectIds){
			return  $http.get("../brand/dele/"+selectIds);
	  }
	  
	  
	  this.search=function(pageNum,pageSize,searchEntity){
		  return $http.post("../brand/search/"+pageNum+"/"+pageSize,searchEntity);
	  }
	
})